### DISCUSSION TITLE
Gephi export issue

### DISCUSSION THREAD

**Discussion by AdodoolaO** [gephi/gephi#2345] [Q&A]
Hi,

When I preview my network on Gephi and I export it, the node names do not appear. The node labels appear instead. I have exported the network in all 3 formats (PDF, PNG SVG). Plss help :)
<img width="1120" alt="Screen Shot 2019-12-06 at 13 00 13" src="https://user-images.githubusercontent.com/58596404/70324802-6033fe00-1828-11ea-8b38-a831ed5a64e4.png">
![Untitled](https://user-images.githubusercontent.com/58596404/70324803-60cc9480-1828-11ea-9100-b4badd35348a.png)

**Comment by tautme** [comment]
In the data laboratory, are the id and label columns the same?
